cmd/docs.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/TouchBistro/goutils/fatal"
+	"github.com/TouchBistro/tb/config"
+	"github.com/spf13/cobra"
+)
+
+var docsCmd = &cobra.Command{
+	Use:   "docs",
```

Comment:
Should add `<service>` so in the docs it's clear service name is a required arg. i.e. `docs <service>`